css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
+  position:absolute;
+  top: 8rem;
+  padding: 120px 0 0 150px;
+  height: 32em;
+  right: -20em;
+  gap: px;
+}
+
+.main-text{
+    width: 50%;
+    grid-area: right;
+    margin: auto;
+    position: absolute;
+    margin-right: 0px;
+    height: 510px;
+  
+}
+.name-title {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+
+}
+.name{
+    margin-top: 20px;
+    height: 20px;
+    font-family: sans-serif;
+    margin-left: 30px;
+}
+.name a{ 
+    line-height: 0.5em;
+    font-size: 20px;
+    margin-left: -20px;
+    
+}
+.title p{
+    font-size: 20px;
+    margin-left: 10px;
+    margin-top: 17px;
+}
+
+
+.blue-button {
+  width: 20px;
+  height: 20px;
+  background-color: #0050FF;
+  margin-top: 20px;
+  margin-left: 14px;
+}
+
+  .grey-box{
+    background-color:#E6DACE;
+    display:block;
+    width: 50%;
+    height: 45em;
+    margin-top: 35px;
+    
+}
+h1{
+position: relative;
+
+}
+
+
+.card{
+    grid-area: left;
+    display: flex;
+    flex-direction: column;
+    background-color: #F4ECE6;
+    right:-20em;
```

Comment:
I think you can drop these negative positions here too. Work on the inside of the card component only.

Imagine the page as like a bento box. https://www.google.com/search?q=bento+box

Your card is like one delicious salad for your bento box. Your card can't know what kind of screen it will show up on, just like your salad doesn't know what bento box it will be spooned into. 🍱 

The page layout rules belong on the page level. Don't try to position each component by hand;  instead define a layout and "pour" each component into the layout section.

Reply:
(This is a complicated but **very useful** concept. Don't feel dismayed if you don't get this straight away! Take this idea to study sessions and buddy calls and think about it together. )